<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>401</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
    <weight>75</weight>
    <italic>true</italic>
    <bold>true</bold>
    <underline>false</underline>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>WhatsThisCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));
}

.QPushButton {
	
	background-color: rgba(135, 135, 135, 200);
}

#pushButton
{
	border-color: rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>90</y>
      <width>641</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>220</y>
      <width>641</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_5">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>130</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_7">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>260</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>3</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Минуты</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>48</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Секунды</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Милисекунды</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Кнопка на клавиатруре</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>110</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Кнопка на мышке</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>48</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Начать</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>240</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Закончить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>330</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>240</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Горячие клавиши</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>48</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Кнопки</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Время повтарения</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>320</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 127);</string>
    </property>
    <property name="text">
     <string>AuxProg</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>АвтоКликер</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
